web/ts/utilities/lecture-hall-validator.ts

```diff
@@ -0,0 +1,12 @@
+export function isLectureHallValid(lectureHall: string): boolean {
+    const regex = /^\d{4}\.[A-Z0-9]{2}\.[A-Z0-9]{3,4}$/;
+    return regex.test(lectureHall);
+}
+
+declare global {
```

Comment:
Why do we need this declare global here? Wouldn't it be enough to just export the function and use it?

Reply:
Theoretically it should suffice, but my local copy would not use this function unless I made it global. I can try making it work without the global declaration if it is an issue.